pkg/controller/disruption/disruption_test.go

```diff
@@ -165,6 +165,37 @@ func newPodDisruptionBudget(t *testing.T, minAvailable intstr.IntOrString) (*pol
 	return pdb, pdbName
 }
 
+func newPodOwnedByRc(t *testing.T, name string, rc *v1.ReplicationController) (*v1.Pod, string) {
```

Comment:
In Go style, we capitalize acronyms: `newPodOwnedByRC`. Same for `RS`, `SS`, etc.